sensor_msgs/JointState goal_state
sensor_msgs/JointState current_state
---
float64 prob
